Gemfile

```diff
@@ -17,4 +17,5 @@ group :test do
   gem 'guard'
   gem 'guard-rspec'
   gem 'listen', '~> 3.0.0'
+  gem 'timecop'
```

Comment:
It would be much better to move timecop to the gemspec instead as a development dependency